src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,123 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  margin: 0 auto;
+  width: maxWidth($card-width, $card-margin, $catalog-xpadding);
```

Comment:
your test falling because you set width to main tag, add ome more wrapper inside and apply width to it